subsys/net/lib/config/init.c

```diff
@@ -60,16 +60,16 @@ static void ipv4_addr_add_handler(struct net_mgmt_event_callback *cb,
 #if defined(CONFIG_NET_DEBUG_APP) && CONFIG_SYS_LOG_NET_LEVEL > 2
```

Comment:
The commit message is too verbose. What about just say: `Use sizeof() for buffer size so that if we change the buffer size, the length of the buffer is automatically set correctly.` or similar. The text inside () is not needed here.

Reply:
I don't see how verbose commit message are bad. The common problem is the opposite ;-).

But will remove text in ( ) as suggested.

Reply:
By verbose I meant the text inside () which was not really that relevant to changes in this commit.